backend/dal/Migrations/Initial/Up/PostUp/02_PIMS_PROPERTY_BOUNDARY_VW.sql

```diff
@@ -0,0 +1,74 @@
+SET XACT_ABORT ON
+GO
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+GO
+BEGIN TRANSACTION
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop view dbo.PIMS_PROPERTY_BOUNDARY_VW
+PRINT N'Drop view dbo.PIMS_PROPERTY_BOUNDARY_VW'
+GO
+DROP VIEW IF EXISTS [dbo].[PIMS_PROPERTY_BOUNDARY_VW]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+PRINT N'Create view dbo.PIMS_PROPERTY_BOUNDARY_VW'
+GO
+CREATE VIEW [dbo].[PIMS_PROPERTY_BOUNDARY_VW] AS
+SELECT PROP.PROPERTY_ID              
+     , PROP.PID                
+     , PROP.PIN                   
+     , PROP.PROPERTY_TYPE_CODE
+     , PROP.PROPERTY_STATUS_TYPE_CODE
+     , PROP.PROPERTY_DATA_SOURCE_TYPE_CODE
+     , PROP.PROPERTY_DATA_SOURCE_EFFECTIVE_DATE
+     , PROP.PROPERTY_CLASSIFICATION_TYPE_CODE
+     , PROP.PROPERTY_TENURE_TYPE_CODE
+     , ADDR.STREET_ADDRESS_1
+     , ADDR.STREET_ADDRESS_2
+     , ADDR.STREET_ADDRESS_3
+     , ADDR.MUNICIPALITY_NAME
+     , ADDR.POSTAL_CODE
+     , PROV.PROVINCE_STATE_CODE
+     , PROV.DESCRIPTION AS PROVINCE_NAME
+     , CNTY.COUNTRY_CODE
+     , CNTY.DESCRIPTION AS COUNTRY_NAME
+     , PROP.NAME
+     , PROP.DESCRIPTION
+     , PROP.ADDRESS_ID                 
+     , PROP.REGION_CODE
+     , PROP.DISTRICT_CODE
+     , PROP.BOUNDARY AS GEOMETRY
+     , PROP.PROPERTY_AREA_UNIT_TYPE_CODE
+     , PROP.LAND_AREA
+     , PROP.LAND_LEGAL_DESCRIPTION
+     , PROP.ENCUMBRANCE_REASON
+     , PROP.IS_SENSITIVE
+     , PROP.IS_OWNED
+     , PROP.IS_PROPERTY_OF_INTEREST
+     , PROP.IS_VISIBLE_TO_OTHER_AGENCIES
+     , PROP.ZONING,ZONING_POTENTIAL
+FROM   PIMS_PROPERTY       PROP                                                    INNER JOIN
+       PIMS_ADDRESS        ADDR ON ADDR.ADDRESS_ID        = PROP.ADDRESS_ID        INNER JOIN
+       PIMS_PROVINCE_STATE PROV ON PROV.PROVINCE_STATE_ID = ADDR.PROVINCE_STATE_ID INNER JOIN
+       PIMS_COUNTRY        CNTY ON CNTY.COUNTRY_ID        = ADDR.COUNTRY_ID
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+COMMIT TRANSACTION
```

Comment:
This is likely to cause failures when running the EF DB Migration.  Have you tested this with a fresh DB?

Reply:
I've tested this with the previous version of this script.

Reply:
trying now with a make db-clean.

Reply:
no issues.